backend/src/main/resources/db/changelog/db.changelog-master.yaml

```
@@ -1758,3 +1758,61 @@ databaseChangeLog:
+        - createView:
+            viewName: test_event_no_phi_view
+            remarks: A subset of the test_event table with columns containing PHI removed.
+            replaceIfExists: true
+            fullDefinition: false
+            selectQuery: SELECT device_specimen_type_id, created_at, created_by, updated_at, updated_by, patient_id, organization_id, device_type_id, result, facility_id, survey_data, date_tested_backdate, test_order_id, correction_status, prior_corrected_test_event_id, internal_id, reason_for_correction FROM ${database.defaultSchemaName}.test_event
+  - changeSet:
+      id: set-no-phi-user-permissions
+      author: josh@skylight.digital
+      comment: Restrict access for the simple_report_no_phi user to certain tables/columns in order to prevent PHI exposure.
+      changes:
+        - sql:
+          sql: |
+            GRANT USAGE ON SCHEMA ${database.defaultSchemaName} TO ${noPhiUsername};
+            GRANT SELECT ON TABLE ${database.defaultSchemaName}.facility_device_type TO ${noPhiUsername};
+            GRANT SELECT ON TABLE ${database.defaultSchemaName}.device_type TO ${noPhiUsername};
+            GRANT SELECT ON TABLE ${database.defaultSchemaName}.organization TO ${noPhiUsername};
+            GRANT SELECT ON TABLE ${database.defaultSchemaName}.patient_answers TO ${noPhiUsername};
+            GRANT SELECT ON TABLE ${database.defaultSchemaName}.facility TO ${noPhiUsername};
+            GRANT SELECT ON TABLE ${database.defaultSchemaName}.patient_link TO ${noPhiUsername};
+            GRANT SELECT ON TABLE ${database.defaultSchemaName}.data_hub_upload TO ${noPhiUsername};
+            GRANT SELECT ON TABLE ${database.defaultSchemaName}.time_of_consent TO ${noPhiUsername};
+            GRANT SELECT ON TABLE ${database.defaultSchemaName}.specimen_type TO ${noPhiUsername};
+            GRANT SELECT ON TABLE ${database.defaultSchemaName}.device_specimen_type TO ${noPhiUsername};
+            GRANT SELECT ON TABLE ${database.defaultSchemaName}.facility_device_specimen_type TO ${noPhiUsername};
+            GRANT SELECT ON TABLE ${database.defaultSchemaName}.test_order TO ${noPhiUsername};
+            GRANT SELECT ON ${database.defaultSchemaName}.api_user_no_phi_view TO ${noPhiUsername};
+            GRANT SELECT ON ${database.defaultSchemaName}.provider_no_phi_view TO ${noPhiUsername};
+            GRANT SELECT ON ${database.defaultSchemaName}.person_no_phi_view TO ${noPhiUsername};
+            GRANT SELECT ON ${database.defaultSchemaName}.test_event_no_phi_view TO ${noPhiUsername};
+      rollback:
+        - sql:
+          sql: |
+            REVOKE ALL PRIVILEGES ON SCHEMA ${database.defaultSchemaName} FROM simple_report_no_phi;
+            REVOKE ALL PRIVILEGES ON ALL TABLES IN SCHEMA ${database.defaultSchemaName};
```

```suggestion
            REVOKE ALL PRIVILEGES ON SCHEMA ${database.defaultSchemaName} FROM ${noPhiUsername};
            REVOKE ALL PRIVILEGES ON ALL TABLES IN SCHEMA ${database.defaultSchemaName} FROM ${noPhiUsername};
```

I think? That last one looked kinda dangerous otherwise.